third_party/timeagojs/timeago.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) 2016 hustcc
+ * License: MIT
+ * Version: v3.0.0
+ * https://github.com/hustcc/timeago.js
+ */
+
+const locales = {};
+
+import {ar} from './timeago-locales';
+import {be} from './timeago-locales';
+import {bg} from './timeago-locales';
+import {ca} from './timeago-locales';
+import {da} from './timeago-locales';
+import {de} from './timeago-locales';
+import {el} from './timeago-locales';
+import {en} from './timeago-locales';
+import {enShort} from './timeago-locales';
+import {es} from './timeago-locales';
+import {eu} from './timeago-locales';
+import {fi} from './timeago-locales';
+import {fr} from './timeago-locales';
+import {he} from './timeago-locales';
+import {hu} from './timeago-locales';
+import {inBG} from './timeago-locales';
+import {inHI} from './timeago-locales';
+import {inID} from './timeago-locales';
+import {it} from './timeago-locales';
+import {ja} from './timeago-locales';
+import {ko} from './timeago-locales';
+import {ml} from './timeago-locales';
+import {nbNO} from './timeago-locales';
+import {nl} from './timeago-locales';
+import {nnNO} from './timeago-locales';
+import {pl} from './timeago-locales';
+import {ptBR} from './timeago-locales';
+import {ro} from './timeago-locales';
+import {ru} from './timeago-locales';
+import {sv} from './timeago-locales';
+import {ta} from './timeago-locales';
+import {th} from './timeago-locales';
+import {tr} from './timeago-locales';
+import {uk} from './timeago-locales';
+import {vi} from './timeago-locales';
+import {zhCN} from './timeago-locales';
+import {zhTW} from './timeago-locales';
+
+// second, minute, hour, day, week, month, year(365 days)
+const SEC_ARRAY = [60, 60, 24, 7, 365 / 7 / 12, 12];
+const SEC_ARRAY_LEN = 6;
+
+// format Date / string / timestamp to Date instance.
+function toDate(input) {
+  if (input instanceof Date) {
+    return input;
+  };
+  if (!isNaN(input)) {
+    return new Date(toInt(input));
+  }
+  if (/^\d+$/.test(input)) {
```

Comment:
This can never be true with the previous `!isNaN` (If this passes, the `!isNaN` will have passed).